src/inference/Inference.jl

```diff
@@ -268,19 +294,35 @@ end
 # Chain making utilities #
 ##########################
 
-function _params_to_array(ts::Vector, spl::Sampler)
+"""
+    getparams(t)
+
+Return a named tuple of parameters.
+"""
+getparams(t) = t.θ
+getparams(t::VarInfo) = tonamedtuple(TypedVarInfo(t))
+
+"""
+    getlp(t)
```

Comment:
maybe `getlogp` for consistency with `VarInfo.logp` 
```suggestion
    getlogp(t)
```

Reply:
I was a bit reluctant of overloading `DynamicPPL.getlogp` here. But maybe we should do that?

Reply:
I think it should be fine to reuse `getlogp` here..

Reply:
I removed `getlp` and instead defined `getlogp` for all transition types.